### ISSUE TITLE
BatchNorm for last conv layer in EAST

### ISSUE THREAD

**Issue by gathierry** [PaddlePaddle/PaddleOCR#2337]
In EAST head, is there still a batch normalization for the last convolution layer? Does that mean output geo map and score map are normalized? 

Does the added batch norm improve the final performance?

Thanks

https://github.com/PaddlePaddle/PaddleOCR/blob/d3a4f788646efd4a3084f5a9807b679177fadfcd/ppocr/modeling/heads/det_east_head.py#L102
